app/models/user.rb

```diff
@@ -1,8 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please make sure the devise is `:confirmable` and asks for email confirmation as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/217550896-bbe61837-62be-4a57-9fda-526bdc06e520.png)